### PR TITLE
new solution

### PR DESCRIPTION
**Pull request by zdlm**
1.remove .idea
2.rebase master

### CODE REVIEW

**Comment by nealstewart** [comment]
@zdlm Hi Leo, please remove your .idea files, and then I will review this PR.

File: code-challenge/src/main/java/com.tradeshift/MessageService.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    public String getContent() {
+        return content;
+    }
+
+    public void setContent(String content) {
+        this.content = content;
+    }
+
+    @Override
```
Comment (nealstewart) [inline code-challenge/src/main/java/com.tradeshift/MessageService.java]: just for fun?

File: code-challenge/src/main/java/com.tradeshift/MessageService.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    public Content(String name) {
+        this.content = "Hello " + name;
+    }
+
+    public String getContent() {
+        return content;
+    }
+
+    public void setContent(String content) {
```
Comment (nealstewart) [inline code-challenge/src/main/java/com.tradeshift/MessageService.java]: Why the setContent accessor?

File: code-challenge/src/main/resources/docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:trusty
+
+EXPOSE 8080
+EXPOSE 9200
+EXPOSE 9300
```
Comment (nealstewart) [inline code-challenge/src/main/resources/docker/Dockerfile]: How are you getting Java on to your docker image?

Comment (zdlm) [inline code-challenge/src/main/resources/docker/Dockerfile]: Sorry, I need to remove it. this is useless. /code-challenge/docker/Dockerfile  is the right one

Comment (nealstewart) [inline code-challenge/src/main/resources/docker/Dockerfile]: Cool. Remove it before I merge this

**Comment by nealstewart** [comment]
Comments made. Ping me when you've replied to/resolved all of the comments.

File: code-challenge/src/main/java/com.tradeshift/MessageService.java (hunk 2 above)
Comment (zdlm) [inline code-challenge/src/main/java/com.tradeshift/MessageService.java]: it is useless ,i will delete it

File: code-challenge/src/main/java/com.tradeshift/MessageService.java (hunk 1 above)
Comment (zdlm) [inline code-challenge/src/main/java/com.tradeshift/MessageService.java]: next iteration it will be a real one

**Comment by nealstewart** [comment]
Rebase that last commit and we're good to go

**Pull request closed without merging**